src/encoding/armor.js

```diff
@@ -360,6 +360,7 @@ export function unarmor(input, config = defaultConfig) {
  * @static
  */
 export function armor(messageType, body, partIndex, partTotal, customComment, config = defaultConfig) {
+  config = { ...defaultConfig, ...config };
```

Comment:
This change was made for [Custom configuration openpgp.revokeKey](https://github.com/openpgpjs/openpgpjs/blob/master/test/general/config.js#L144) test to pass. `defaultConfig` is merged in the `revokeKey` method but `armor` is no longer called from `revoke` method's context. Since `defaultConfig` is not merged in [PrivateKey.armor](https://github.com/openpgpjs/openpgpjs/blob/master/src/key/private_key.js#L76), I thought it might be good to place this merge logic in the armor encoder to avoid something like this (a missed default config merge) in the future.

I mention this because it might be better to place this line in the `PrivateKey.armor` method instead to avoid some unnecessary merging.